databricks/koalas/base.py

```diff
@@ -184,8 +184,14 @@ def __sub__(self, other):
             return _column_op(spark.Column.__sub__)(self, other)
 
     __mul__ = _column_op(spark.Column.__mul__)
-    __div__ = _numpy_column_op(spark.Column.__div__)
```

Comment:
FYI: In Python 3.x, `__div__` and `__rdiv__` no more supported. Using `__truediv__` and `__rtruediv__`instead.